input/docs/handbook/collections/index.md

```diff
@@ -82,14 +82,79 @@ var myBindingOperation = mySource
 
 The API for the above is the same for cache and list.
 
-## So what's the difference between an observable list and an observable cache
+## So what's the difference between a SourceList and a SourceCache
 
-I get asked this question a lot and the answer is really simple.  If you have a unique id, you should use
-an observable cache as it is dictionary based which will ensure no duplicates can be added and it notifies on adds, updates and removes, whereas list allows duplicates and only has no concept of an update.
+If you have a unique id, you should use an observable cache as it is dictionary based which will ensure no duplicates can be added and it notifies on adds, updates and removes, whereas list allows duplicates and only has no concept of an update. `SourceCache` has several performance advantages over `SourceList`, so if possible, always prefer `SourceCache` over `SourceList`.
 
 There is another difference. The cache side of dynamic data is much more mature and has a wider range of operators. Having more operators is mainly because I found it easier to achieve good all round performance with the key based operators and do not want to add anything to Dynamic Data which inherently has poor performance.
 
-## How to track changes in collections of Reactive Objects?
+# Using DynamicData with ReactiveUI
+
+When building applications with ReactiveUI and DynamicData, you have a choice to work with mutable or with immutable collections. When working with immutable ones, using an `ObservableAsPropertyHelper<T>` is enough in simple cases. The `ObservableAsPropertyHelper<T>` represents an `Observable<T>`, a stream of values over time. You can treat those values as events, and the new values as event arguments. This means if you are using immutable collections, you can treat them as event arguments and update a property with a new collection each time it changes. See [ObservableAsPropertyHelper Handbook section](/docs/handbook/oaph/) to learn how to use this feature. Note, that creating a new colletion for each update degrades performance and should be generally avoided, instead, use DynamicData.
+
+### An Example
+
+Imagine your application needs a service that will expose a collection mutated by a background worker. You need to get change notifications from it somehow to synchronize it with the user interface. Here DynamicData comes to the rescue. You expose an `IObservableList<T>` (or even better an `IObservableCache<TObject, TKey>`) to the outer world, and DynaicData takes case of allowing you to observe their changes. 
+
+```cs
+public class Service 
+{
+    private readonly SourceList<bool> _items = new SourceList<bool>();
+
+    public IObservable<IChangeSet<bool>> Items { get; }
```

Comment:
This could cause issues if you have multiple subscribers to the `Connect()` which isn't the case here.

If the SourceList changes between the time the Connect() is called and subscription happens, it could be out of sync. It's why in most of Roland's example's he tends to do the `Publish()` then later `Connect()` if he's going to have multiple subscribers. `Publish()` creates a Subject() after first subscription. https://github.com/dotnet/reactive/blob/0f837d11385cfaf575861ccc5a5fbcafb22d888f/Rx.NET/Source/src/System.Reactive/Subjects/ConnectableObservable.cs#L42

Reply:
Hmm.. I'm researching I might be wrong about part of that.

Reply:
I agree with @glennawatson and I would be wary about setting a property of `dataSource.Connect()`.  The reason is that each time `dataSource.Connect()` is called, a new observable is created. Semantically it is not clear what the value of the property means.